flocker/node/agents/blockdevice.py

```diff
@@ -956,7 +956,7 @@ class MandatoryProfiles(Values):
     GOLD = ValueConstant(u'gold')
     SILVER = ValueConstant(u'silver')
     BRONZE = ValueConstant(u'bronze')
-    DEFAULT = SILVER
+    DEFAULT = ValueConstant(BRONZE.value)
```

Comment:
Thanks for the note, @wallnerryan ! In the interest of moving ahead, will address ``default`` handling in a follow up branch. Please let me know if that does not sound reasonable. Thanks!